package.json

```diff
@@ -98,6 +98,7 @@
     ]
   },
   "dependencies": {
+    "prop-types": "^15.5.8",
     "react": "^15.0.0"
```

Comment:
@bvellek Should we change this to `^15.3.0`?  Looking at this documentation https://www.npmjs.com/package/prop-types#react-15